tests/checkandmutaterow_test.go

```diff
@@ -296,6 +297,10 @@ func TestCheckAndMutateRow_Generic_CloseClient(t *testing.T) {
 	// 4b. Check that all the batch-one requests succeeded
 	checkResultOkStatus(t, resultsBatchOne...)
 	for i := 0; i < halfBatchSize; i++ {
+		assert.NotNil(t, resultsBatchOne[i].Result)
```

Comment:
There are other *_test.go files that may need this too. You may apply it to all of them for consistency?